### DISCUSSION TITLE
New app idea "mouse jiggler"

### DISCUSSION THREAD

**Discussion by tberghuis** [AppGround-io/bluetooth-keyboard-and-mouse-support#255] [Ideas]
see https://www.amazon.com.au/Automatic-Computer-Simulate-Movement-Prevents/dp/B08TQQJVJ5
Was going to try to build it myself but am giving up as can't find good working examples of code using BluetoothHidDevice

**Comment by ianmperryman** [comment]
This would be a useful feature for those of us with laptops managed by an employer and prevented from changing "sleep" time to something reasonable.
